client-v2/src/shared/types/Collection.ts

```diff
@@ -8,8 +8,10 @@ interface Group {
   articleFragments: string[];
 }
 
-type Stages = 'draft' | 'live';
+// CollectionItemSets represent all of the lists of collectionItems available in a collection.
```

Comment:
I've added this here as I'm not convinced developers will find it in a readme when they most need it, but happy to move it if anyone else has strong opinions.